content/departments/product-engineering/engineering/enablement/dev-experience/index.md

```diff
@@ -63,11 +66,11 @@ The DevX team currently has weekly sync meetings and biweekly retrospectives wit
 - [Team sync notes](https://docs.google.com/document/d/1Lm6GT-F4v9OTa5wxa1-AKLtNwlDkORbbeGjqVd9kWPg/edit)
 - [Retrospective notes](https://docs.google.com/document/d/1QR1It6KGccwWRpASH16J64QNkpHtngI4o2ttrGpVCwU/edit#)
 
-#### `sg` hack hour
+#### `sg` hacking hour
```

Comment:
I think `hack hour` is more grammatically correct? 🤔 anecdotally a search for "hack hour" comes up with:

![image](https://user-images.githubusercontent.com/23356519/149376773-86a095b7-afdf-497b-8a24-bdb15f309eb7.png)

while "hacking hour" doesn't come up with any other usages

that said this seems a more rare term than I expected, but intuitively "hack hour" sounds more right to me

Reply:
I'm struggling with other verbs though - `run hour` sounds just as weird as `running hour` 😅

Reply:
It's just that's how it's named in the calendar and announced on Slack, so I went with it without thinking too much about it :